input/resources/au-core-responder.xml

```diff
@@ -3205,7 +3213,7 @@
 </extension>
 <type value="Encounter"/>
 <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-encounter"/>
-<documentation value="If the responder supports the Encounter resource, the responder **SHALL** support the AU Core profile and the conformance expectations for the Encounter resource."/>
+<documentation value="If the responder supports the Encounter resource, the responder **SHALL** support the AU Core profile and the conformance expectations for the Encounter resource.&#xA;&#xA;The Encounter resource can communicate the clinical indication using either a code in Encounter.reasonCode or a reference using Encounter.reasonReference. Although both `Encounter.reasonCode` and `Encounter.reasonReference` are marked as Must Support, a responder is not required to support both but **SHALL** support at least one of these elements. The requester**SHALL** support both elements."/>
```

Comment:
Apply formatting to all instances of Encounter.reasonCode and Encounter.reasonReference.

Reply:
Fix not included in this PR.